src/OrbitQt/CMakeLists.txt

```diff
@@ -112,9 +114,9 @@ target_link_libraries(
           SyntaxHighlighter
           qtpropertybrowser::qtpropertybrowser)
 
-if(WITH_CRASH_HANDLING)
-  target_link_libraries(OrbitQt PRIVATE CrashHandler)
-  target_compile_definitions(OrbitQt PRIVATE ORBIT_CRASH_HANDLING)
+if(WITH_CRASH_HANDLING)
+  target_link_libraries(OrbitQt PRIVATE CrashHandler)
+  target_compile_definitions(OrbitQt PRIVATE ORBIT_CRASH_HANDLING)
```

Comment:
What has changed here?

Reply:
Took me a while to figure it out. I think vscode or clang format fixed line endings here (CRLF -> LF). 
![Screenshot 2021-03-11 at 10 49 37](https://user-images.githubusercontent.com/7501835/110768382-7adcfa00-8257-11eb-89e1-f4dcc3619efe.png)

Reply:
So nothing changed and it can be removed from this commit?

Reply:
Well. The file contained mixed line endings before this commit. Which is a bad thing in the first place. Now its just LF line endings and thats a good thing. I could add a dedicated "Fix line endings" commit, but for the sake of simplicity, I would leave it as is. wdyt?

Reply:
I personally value clean commits a lot higher than consistent line endings, but I also don't feel strongly at all about this. So if you prefer to leave it, that's fine with me!